lab_1/main.py

```diff
@@ -0,0 +1,43 @@
+"""
+Labour work #1
+Count frequencies dictionary by the given arbitrary text
+"""
+
+
+def calculate_frequences(text: str):
+    alph = '" "abcdefghijklmnopqrstuvwxyz'

```

Comment:
@LidaVaskina no need for " " in alph, just leave a space without quotes with this approach.